lahendus/login/messages/messages_et.properties

```diff
@@ -47,6 +48,12 @@ proceedWithAction=Kinnita &raquo;
 emailVerifiedMessage=Meiliaadress kinnitatud.
 doClickHere=Vajuta siia
 
+doTryAgain=Proovi uuesti
+doTryAnotherWay=Proovi teisiti
+
+loginChooseAuthenticator=Vali sobiv autentimise viis
```

Comment:
What are these for - when are they used?

Reply:
@PriitPaluoja TODO: make sure these messages make sense and are as useful as possible.